src/main/resources/interface-specification.yml

```diff
@@ -6873,6 +6873,80 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  /clients/consumer:
```

Comment:
unfortunately this would clash with `/clients/{clientId}` endpoint, that's why in the authorization process we choose the ugly `/clientsConsumer`.
If there are no better ideas today (which are definitely welcome), I would stick with that endpoint